### PR TITLE
add --version to pkgmt setup

### LINKED ISSUE #72: add `--version` to `pkgmt setup`
should allow selecting a python version. the implementation is already there, we just have to modify the CLI

to test it, we can add a test to call the cli with --version and then mock the function and ensure it gets called with the right argument

### PR DESCRIPTION
**Pull request by bryannho**
Changes:
- Modified pkgmt setup to include a "--version" flag

Closes #72 

Checklist before requesting a review
- [x] Performed a self-review of my code
- [x] Formatted my code with [pkgmt format](https://ploomber-contributing.readthedocs.io/en/latest/contributing/pr.html#linting-formatting)
- [x] Added tests

### CODE REVIEW

File: src/pkgmt/cli.py [hunk 1]
```diff
@@ -138,7 +138,13 @@ def release(tag, production, yes):
     default=False,
     help="Install documentation dependencies",
 )
-def setup(doc):
+@click.option(
+    "--version",
+    default=None,
+    type=str,
+    help="Select a Python version, default is 3.10" "Eg: --version 3.9",
```
Comment (edublancas) [inline src/pkgmt/cli.py:145]:
```suggestion
    help="Select a Python version, default is 3.10,
```

File: tests/test_cli.py [hunk 2]
```diff
@@ -262,3 +263,33 @@ def test_lint_black_pyproj(pyproject, command, output, tmp_empty):
...
+        [
+            ["setup"],
+            None,
+            False,
+        ],
+        [
+            ["setup", "--version", "3.9"],
+            "3.9",
+            False,
+        ],
```
Comment (edublancas) [inline tests/test_cli.py:280]: can you test with other python version values?

**Review by edublancas — CHANGES_REQUESTED** [review]

File: src/pkgmt/cli.py (hunk 1 above)
Comment (bryannho) [reply src/pkgmt/cli.py]: [Fixed](https://github.com/ploomber/pkgmt/pull/74/files#diff-71611c60a29b6d5c2191fb918506645b99ede87211e904c01553bd45af08c8e8R145)

File: tests/test_cli.py (hunk 2 above)
Comment (bryannho) [reply tests/test_cli.py]: [Fixed here](https://github.com/ploomber/pkgmt/pull/74/files#diff-4e8715c7a425ee52e74b7df4d34efd32e8c92f3e60bd51bc2e1ad5943b82032eR268-R280) - Added more version values to test

**Review by edublancas — APPROVED** [review]

**Pull request merged**